apps/web/src/modules/job/apply/pages/hooks/index.ts

```diff
@@ -0,0 +1,67 @@
+import { GetJobDto, GetResumesDto, ResumeDto, ResumeIdDto } from '@modela/dtos'
+import { useSnackbar } from 'common/context/SnackbarContext'
+import { useErrorHandler } from 'common/hooks/useErrorHandler'
+import { apiClient } from 'common/utils/api'
+import { useRouter } from 'next/router'
+import React from 'react'
+import { useCallback, useEffect, useState } from 'react'
+
+const useResume = () => {
+  const router = useRouter()
+  const { jobId } = router.query
+  const { handleError } = useErrorHandler()
+  const [loading, setLoading] = useState(false)
+  const [jobTitle, setJobTitle] = useState('')
+  const { displaySnackbar } = useSnackbar()
+
+  const [Id, setId] = React.useState<number | undefined>()
+
+  const handleSuccess = useCallback(async () => {
+    setLoading(true)
+    try {
+      const resumeIdBody: ResumeIdDto = {
```

Comment:
resumeId should be checked before POST to backend (we should not send bad request data)